versioned_docs/version-24.04/administration/centreon-ha/operating-guide.md

```diff
@@ -0,0 +1,821 @@
+
+The Resource Status page gives you the following information:
+
+* On both central nodes, the **PCS-Status** service gives you the detailed state of the cluster. The output of the service in the details panel is the output of the `pcs status` command.
+* You can know which central node is the active node by looking at which node is carrying the cluster resources in the output of the **PCS-Status** service on each central.
+
+## Remove an error displayed in the cluster status
+
+Once the cause of the error has been identified and fixed (see the [troubleshooting guide](troubleshooting-guide.md)), you must delete the error message manually:
+
+```bash
+pcs resource cleanup
+```
+
+Or, if you want to remove only the errors linked to one resource:
+
+```bash
+pcs resource cleanup <resource_name>
+```
+
+## Check the constraints
+
+If a failover has occurred at some point, there may be some leftover location constraints. Run the following command to display the current constraints:
+
+```bash
+pcs constraint
+```
+
+The command should return this:
+
+```text
+Location Constraints:
+  Resource: cbd_rrd-clone
+    Disabled on:
+      Node: @DATABASE_NODE1_NAME@ (score:-INFINITY)
+      Node: @DATABASE_NODE2_NAME@ (score:-INFINITY)
+  Resource: centreon
+    Disabled on:
+      Node: @DATABASE_NODE1_NAME@ (score:-INFINITY)
+      Node: @DATABASE_NODE2_NAME@ (score:-INFINITY)
+  Resource: ms_mysql-clone
+    Disabled on:
+      Node: @CENTRAL_NODE1_NAME@ (score:-INFINITY)
+      Node: @CENTRAL_NODE2_NAME@ (score:-INFINITY)
+  Resource: php-clone
+    Disabled on:
+      Node: @DATABASE_NODE1_NAME@ (score:-INFINITY)
+      Node: @DATABASE_NODE2_NAME@ (score:-INFINITY)
+Ordering Constraints:
+Colocation Constraints:
+  vip_mysql with ms_mysql-clone (score:INFINITY) (rsc-role:Started) (with-rsc-role:Master)
+  ms_mysql-clone with vip_mysql (score:INFINITY) (rsc-role:Master) (with-rsc-role:Started)
+Ticket Constraints:
+
+```
+
+The output shows the constraints you have defined during the installation procedure: the **ms_mysql-clone** resource only runs on the database nodes, the **cbd_rrd-clone**, **centreon** and **php-clone** resources only run on the central nodes.
+
+To remove unwanted constraints, run the following command:
+
+```bash
+pcs resource clear centreon
+```
+
+## Check the status of the database synchronization
+
+To check that the database synchronization is working, run the following command:
+
+```bash
+/usr/share/centreon-ha/bin/mysql-check-status.sh
+```
+
+The command should return the following information:
+
+```text
+Connection MASTER Status '@DATABASE_NODE1_NAME@' [OK]
+Connection SLAVE Status '@DATABASE_NODE2_NAME@' [OK]
+Slave Thread Status [OK]
+Position Status [OK]
+```
+
+If the synchronization shows `KO`, you must fix it. The procedure below explains how to manually re-enable MariaDB replication.
+
+### Restore MariaDB master-slave replication
```

Comment:
We probably want to replace master and slave with active and passive in this section (since we do that in the other parts of the doc)

Reply:
very true. thanks